beacon-chain/core/state/state_transition.go

```diff
@@ -128,47 +128,50 @@ func NewStateTransition(
 // the proposed shard block with latest beacon chain slot numbers.
 func crossLinkCalculations(
 	st *types.BeaconState,
-	pendingAttestations []*pb.AggregatedAttestation,
+	pendingAttestations []*pb.PendingAttestationRecord,
 	currentSlot uint64,
 ) ([]*pb.CrosslinkRecord, error) {
 	slot := st.LastStateRecalculationSlot() + params.BeaconConfig().CycleLength
 	crossLinkRecords := st.LatestCrosslinks()
-	for _, attestation := range pendingAttestations {
-		shardCommittees, err := v.GetShardAndCommitteesForSlot(
-			st.ShardAndCommitteesForSlots(),
-			st.LastStateRecalculationSlot(),
-			attestation.GetSlot(),
-		)
-		if err != nil {
-			return nil, err
-		}
-
-		indices, err := v.AttesterIndices(shardCommittees, attestation)
-		if err != nil {
-			return nil, err
-		}
-
-		totalBalance, voteBalance, err := v.VotedBalanceInAttestation(st.ValidatorRegistry(), indices, attestation)
-		if err != nil {
-			return nil, err
-		}
-
-		newValidatorSet, err := incentives.ApplyCrosslinkRewardsAndPenalties(
-			crossLinkRecords,
-			currentSlot,
-			indices,
-			attestation,
-			st.ValidatorRegistry(),
-			v.TotalActiveValidatorDeposit(st.ValidatorRegistry()),
-			totalBalance,
-			voteBalance,
-		)
-		if err != nil {
-			return nil, err
-		}
-		st.SetValidatorRegistry(newValidatorSet)
-		crossLinkRecords = UpdateLatestCrosslinks(slot, voteBalance, totalBalance, attestation, crossLinkRecords)
-	}
+	// TODO(#781): Pending refactor from the spec.
+	_ = slot
+	_ = crossLinkRecords
+	// for _, attestation := range pendingAttestations {
```

Comment:
Not entirely sure what to do here other than commenting out the bulk of this function, as it will break and the inner logic will be inherently different due to spec changes. Ideas @terenc3t @prestonvanloon?